exporters/cinder.go

```diff
@@ -308,7 +312,52 @@ func ListVolumeLimits(exporter *BaseOpenStackExporter, ch chan<- prometheus.Metr
 		if err != nil {
 			return err
 		}
-
+		jsonLimits := quotasets.GetUsage(exporter.Client, p.ID).PrettyPrintJSON()
+		var jsonParse map[string]interface{}
+		_ = json.Unmarshal([]byte(jsonLimits), &jsonParse)
+		quota := jsonParse["quota_set"].(map[string]interface{})
```

Comment:
quota, ok := 
if !ok { return fmt.Errorf() }

Reply:
fixed in 085a1ae20af7403bcf732e4dc0d2089c129747f3